### PR TITLE
ADEN-7294 New implementation of InstartLogic recovery

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexus.js [hunk 1]
```diff
@@ -65,8 +65,8 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.appnexus',[
 		return adContext.get('bidders.appnexus');
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexus.js]: Does it mean that AN is going to be enabled both for BT and IL rec? Shouldn't we enable it based on `context.opts.wadIL`?

File: extensions/wikia/AdEngine/js/wad/ilRecLoader.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+			isAsync: false,
+			node: doc.head.lastChild
+		});
+	}
+
+	function init() {
+		if (adContext.get('opts.babRec')) {
+			doc.addEventListener('bab.blocking', injectScript);
+		} else {
+			injectScript();
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/wad/ilRecLoader.js]: Same question here - do we really want to inject this script without babDetection?

Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/wad/ilRecLoader.js]: And same answer: It's just an option. We had that before, why not leave it after cleanup-o-refactor. Waiting for grooming - we probably gonna remove that.

Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/wad/ilRecLoader.js]: Ok, removing.

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexus.js (hunk 1 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexus.js]: That's true, condition expanded.

**Review by fraszczakszymon — APPROVED** [review]

**Review by fraszczakszymon — APPROVED** [review]

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**